python/src/aiconfig/editor/client/src/components/EditorContainer.tsx

```diff
@@ -79,9 +79,9 @@ export type AIConfigCallbacks = {
   deletePrompt: (promptName: string) => Promise<void>;
   getModels: (search: string) => Promise<string[]>;
   getServerStatus?: () => Promise<{ status: "OK" | "ERROR" }>;
-  runPrompt: (promptName: string) => Promise<{ aiconfig: AIConfig }>;
-  runPromptStream: (
+  runPrompt: (
     promptName: string,
+    enableStreaming: boolean,
     onStream: RunPromptStreamCallback
   ) => Promise<void>;
```

Comment:
Let's make this return `Promise<{aiconfig?: AIConfig}>;` so that external implementations don't necessarily need to implement their execution via streaming